lib/vanagon/component/dsl.rb

```diff
@@ -276,8 +276,25 @@ def url(the_url)
       # Sets the md5 sum to verify the sum of the source
       #
       # @param md5 [String] md5 sum of the source for verification
+      # md5 is our default
       def md5sum(md5)
         @component.options[:sum] = md5
+        @component.options[:sum_type] = "md5"
+      end
```

Comment:
Should we expose corresponding `#sha256sum` and `#sha512sum` methods? And if we should, should we not expose the `#sum` and `#sum_type` methods?